bson/json_util.py

```diff
@@ -229,7 +229,7 @@ class JSONOptions(CodecOptions):
       - `tz_aware`: If ``True``, MongoDB Extended JSON's *Strict mode* type
         `Date` will be decoded to timezone aware instances of
         :class:`datetime.datetime`. Otherwise they will be naive. Defaults
-        to ``True``.
+        to ``False``.
```

Comment:
Please add a `.. versionchanged:: 4.0` to the end of this docstring with a description of the change.

Reply:
Done.